zc_plugins/DisplayLogs/v3.0.2/admin/includes/languages/english/lang.display_logs.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @copyright Copyright 2003-2022 Zen Cart Development Team
+ * @copyright Portions Copyright 2003 osCommerce
+ * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
+ * @version $Id: Steve 2021 Apr 02 New in v1.5.8-alpha $
+*/
+
+
+$define = [
+    'HEADING_TITLE' => 'Display Debug Log Files',
+    'TABLE_HEADING_FILENAME' => 'File Name',
+    'TABLE_HEADING_MODIFIED' => 'Date Modified',
+    'TABLE_HEADING_FILESIZE' => 'File Size (bytes)',
+    'TABLE_HEADING_DELETE' => 'Delete?',
+    'TABLE_HEADING_ACTION' => 'Action',
+    'BUTTON_INVERT_SELECTED' => 'Inverse Selection(s)',
```

Comment:
Should be Invert (verb), not Inverse (adjective).

Reply:
True, noting that a change will be required in the overall instructional text as well.  Will you PR?

Reply:
I had wanted to use `inverse of selection(s)` but that seemed too wordy.  I had also toyed with `invert selection(s)` but that could mean to put the items currently at the top of the list at the bottom. 

So, next alternate text suggestion as there is issue with the current?

Reply:
I did this

![2023-01-15 15_11_19-Admin Display Logs — Mozilla Firefox](https://user-images.githubusercontent.com/4391026/212545747-036c315a-1bb4-469c-a3bd-b735baae5c80.gif)

I've rejigged the bit about user prefixes so it doesn't look so bad, until someone else fixes that properly/as they see fit.

Reply:
How about either "Swap Selections" or "Switch Selections"?  It might help to give an example in the unordered list, e.g.

```
For example, if you want to delete all but one file, tick the selection for the file to be kept, then "Swap Selections" and finally "Delete Selected".
```

Reply:
To be honest, I was all for throwing away the explanation completely...are Zen Cart admins really that dumb?
The only non-intuitive info is that which points out that the listing is derived from pre-set and user-defined prefixes.
The rest is bleeding obvious...https://www.youtube.com/watch?v=iJSyGlmT3lk